Sources/Core/Shared/ComposedOperation.swift

```diff
@@ -18,7 +18,7 @@ public class ComposedOperation<T: Operation>: Operation {
     }
 
     init(operation composed: T) {
-        self.target = composed as? Operation ?? GroupOperation(operations: [composed])
+        self.target = composed ?? GroupOperation(operations: [composed])
```

Comment:
@fcobia - this is actually a mistake though - on my original work in the `swift/3.0` branch.

The type constraint should be `Foundation.Operation` and the conditional cast should be `Operations.Operation`.